edge/discussion_materials/ONNX Edge scope and profile definition.md

```diff
@@ -0,0 +1,72 @@
+
+#### Size
+
+Size here refers to both the model on-disk storage size and the memory that the runtime reqiures to run the inference. In ONNX 1.4 release, support for large models (larger than 2GB) and store the data externally is added as a new feature. But in edge device, we do not want such huge models running and consuming the precious memory. There will always be compromises between performance, speed, cost and consumption. There are many ways to compress our models while keeping the accuracy like changing to quantized ops, model compression etc. 
+
+#### Latency
+
+In many scenarios, fast processing speed and low latency are required. Vendor-specific runtime optimization are encouraged while we should also keep our optimization work going. works on the speed over laps with some of the work on the bandwidth, since a broader bandwidth always provide less latency thus improving overall speed.
+
+#### Power consumption
+
+Power consumption is critical for an edge device. Methods are to be defined to reduce the power consumption/extend the service time of this edge device while maintaining the same level of performance:
+
+1. A larger battery which extend the servie time of an edge device.
+
+2. A better chipset which utilized with more advanced algorithms and technique to lower consumption in every clock time.
+
+3. New scheduling system to switch off some unused applicatio or parts
+
+4. Better cooling system that doesn't consume like the old fashioned fan
+
+5. etc. 
+
+
+#### Data locality
+
+Depending on the application, in particular on the required data type and format, it is up to the developer to select the best transmission technology. Sometimes, it is possible to create a local network and to send the data to the cloud for further processing or for being stored. However, future IoT nodes will heavily depend on cellular communication, e.g. 5G technology.
+Attacks can involve sensors nodes to collect privacy data from users, which could be used for analysis purposes or to profile users, or involve in auto pilot cars, healthcare devices (like smart watch) or literally every electrical item that will potentially be equipped with a network access. 
+Edge devices will be built with hardware cryptographic accelerators for optimizing power consumption and latency of data transmission. These hardware accelerators can (and should) also be used for blocking the basic IP stealing attempts, encrypting the data for a better security.
```

Comment:
This paragraph seems out of context/ speculative. Using "will be built" seems as too strong. Plus what is meaning of "hardware cryptographic accelerators for optimizing power consumption and latency of data transmission." ? Kind of reads as: cryptography is needed to save power and improve transmission latency...